app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_user!
+      def index
+        user = User.find(params[:user_id])
+        render_paginated user.rents, each_serializer: RentSerializer
+      end
+
+      def create
+        user = User.find(params[:user_id])
```

Comment:
Same here

Reply:
The same reasons that I've exposed previously.